website/docs/r/kubernetes_cluster.html.markdown

```diff
@@ -670,6 +672,20 @@ A `windows_profile` block supports the following:
 
 ---
 
+A `http_proxy_config` block supports the following:
+
+* `http_proxy` - (Optional) The proxy address to be used when communicating over HTTP.
+
+* `https_proxy` - (Optional) The proxy address to be used when communicating over HTTPS.
+
+* `no_proxy` - (Optional) The list of domains that will not use the proxy for communication.
+
+-> **Note:** If you specify the `vnet_subnet_id`, be sure to include the CIDR in the no_proxy list.
```

Comment:
Is this what you mean? Would be nice if it is possible to link to the default list as well from here.
```suggestion
-> **Note:** If you specify `default_node_pool.0.vnet_subnet_id`, be sure to include the Subnet CIDR in the `no_proxy` list. 
```